apps/crn-frontend/src/discover/Routes.tsx

```diff
@@ -2,7 +2,8 @@ import { FC, lazy, useEffect } from 'react';
 import { Switch, Route, useRouteMatch, Redirect } from 'react-router-dom';
 import { Frame } from '@asap-hub/frontend-utils';
 import { discover } from '@asap-hub/routing';
-import { DiscoverPage } from '@asap-hub/react-components';
+import { DiscoverPage, TutorialsPage } from '@asap-hub/react-components';
```

Comment:
Should the discover page still be called the discover page?

Reply:
It should probably be changed. There are quite a few discover references in the codebase and I thought having a separate pr to refactor those might be better? The current route is /guides-tutorials so following the pattern DiscoverPage should become GuidesTutorialsPage